src/features/game/types/collectibles.ts

```diff
@@ -221,6 +228,65 @@ export const HELIOS_BLACKSMITH_ITEMS: (
       Orange: new Decimal(10),
     },
   },
+  "Stone Beetle": {
+    description: "",
+    boost: "+0.1 Stone",
+    coins: 1000,
+    ingredients: {
+      Stone: new Decimal(20),
+    },
+  },
+  "Iron Beetle": {
+    description: "",
+    boost: "+0.1 Stone",
+    coins: 2000,
+    ingredients: {
+      Iron: new Decimal(20),
+    },
+  },
+  "Gold Beetle": {
+    description: "",
+    boost: "+0.1 Stone",
+    coins: 10000,
+    ingredients: {
+      Gold: new Decimal(20),
+    },
+  },
+  "Fairy Circle": {
+    description: "",
+    boost: "+0.2 Mushroom",
```

Comment:
Should probably be explicit (`Wild Mushroom`) since it presumably doesn't impact `Magic Mushroom`.